### PR TITLE
updated design toolkits

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by PRMerger17** [comment]
@alur2020 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by mcleanbyron** [comment]
@Karl-Bridge-Microsoft or @jwmsft, can either of you review this one?

File: hub/apps/fluent-design-system.md [hunk 1]
```diff
@@ -247,7 +247,7 @@ Add material to your UWP app:
 
 ## Design toolkits and code samples
 
-Want to get started creating your own apps with Fluent Design? Our toolkits for Adobe XD, Adobe Illustrator, Adobe Photoshop, Framer, and Sketch will help jumpstart your designs, and our samples will help get you coding faster.
+Want to get started creating your own apps with Fluent Design? Our toolkits for Figma, Adobe XD and Sketch will help jumpstart your designs, and our samples will help get you coding faster.
```
Comment (Karl-Bridge-Microsoft) [inline hub/apps/fluent-design-system.md]: Oxford comma? :-)

Comment (ghost) [reply hub/apps/fluent-design-system.md]: Didn't know that I had use a oxford comma haha

**Comment by PRMerger16** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: hub/apps/fluent-design-system.md (hunk 1 above)
Comment (ktoliver) [reply hub/apps/fluent-design-system.md]: I added the comma!

**Comment by mcleanbyron** [comment]
@hickeys - Can you please review the toolkit info in this edit?

**Comment by hickeys** [comment]
@alur2020 - Can you comment on your thinking a little more here? The download page has packages for Figma, Sketch, and XD, and does not offer downloads for Illustrator, Photoshop, or Framer. Was there another edit that you intended to include in this PR?

**Comment by ghost** [comment]
No. The original text said that there are design toolkits for Ilustrator, Photoshop or Framer. I updated the file because these design toolkits aren't available anymore

Pobierz aplikację Outlook dla systemu Android<https://aka.ms/ghei36>

________________________________

**Comment by hickeys** [comment]
Oh, wow. That's embarrassing! Thanks so much for the PR.

**Review by hickeys — APPROVED** [review]

**Pull request merged**